composer.json

```diff
@@ -19,15 +19,16 @@
 		"bin/yaml-sort-checker"
 	],
 	"require": {
-		"php": "~7.2",
+		"php": "~7.2 | ^8.0",
 		"symfony/console": "~3.4|~4.3|~5.0",
 		"symfony/yaml": "~3.4|~4.3|~5.0"
 	},
 	"require-dev": {
-		"consistence/coding-standard": "3.9",
-		"jakub-onderka/php-parallel-lint": "1.0.0",
-		"phpstan/phpstan-shim": "0.11.19",
-		"phpunit/phpunit": "8.4.3"
+		"consistence/coding-standard": "^3.9",
+		"php-parallel-lint/php-parallel-lint": "^1.0.0",
+		"phpstan/phpstan": "0.12.57",
+		"phpunit/phpunit": "^8.4.3 | ^9.0",
```

Comment:
If the support for PHP 7.2 is dropped, the latest PHPUnit should be compatible with all supported versions

Reply:
Changed to just PHPUnit 9